nbs/05_methods.base.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            x: jnp.ndarray, # Input to be explained
</pre><p>annotate as <code>x: Array</code> from now on </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/ReLax/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>